packages/okta-react-native/package.json

```diff
@@ -1,59 +1,44 @@
 {
   "name": "@okta/okta-react-native",
-  "version": "0.1.2",
-  "description": "React Native support for OpenID Connect",
-  "main": "src/token-client.js",
+  "title": "React Native Okta Sdk Bridge",
+  "version": "1.0.0",
+  "description": "Okta OIDC for React Native",
+  "main": "index.js",
   "scripts": {
-    "test": "yarn test:unit",
-    "test:unit": "jest --coverage --verbose --testPathPattern __tests__/.*.test.js"
-  },
-  "jest": {
-    "browser": true,
-    "preset": "jest-expo",
-    "automock": false,
-    "setupFiles": [
-      "./setupJest.js"
-    ],
-    "transformIgnorePatterns": [
-      "node_modules/(?!(jest-)?react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|react-navigation|@react-navigation/.*|@unimodules/.*|sentry-expo|native-base)"
-    ]
+    "test": "echo \"Error: no test specified\"",
+    "test:debug": "echo \"Error: no test specified\""
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/okta/okta-oidc-js.git"
   },
-  "license": "Apache-2.0",
   "files": [
-    "src/"
+    "index.js",
+    "android",
+    "ios",
+    "LICENSE",
+    "CHANGELOG.md",
+    "package.json"
   ],
-  "dependencies": {
-    "@okta/configuration-validation": "^0.3.0",
-    "b64u-lite": "^1.0.1",
-    "dlv": "^1.1.2",
-    "expo-constants": "^4.0.0",
-    "expo-secure-store": "^4.0.0",
-    "expo-web-browser": "^4.0.0",
-    "isomorphic-webcrypto": "^1.6.1",
-    "jwt-lite": "^1.1.2",
-    "str2buf": "^1.3.0"
-  },
+  "keywords": [
+    "react",
+    "react-native",
+    "authentication",
+    "okta",
+    "oidc"
+  ],
+  "license": "Apache-2.0",
+  "readmeFilename": "README.md",
   "peerDependencies": {
-    "react-native": "*",
-    "react-native-securerandom": "^0.1.1",
-    "react-native-unimodules": "0.3.1"
+    "react": "^16.5.0",
+    "react-native": ">=0.57.0-rc.0 <1.0.x"
```

Comment:
A 1.0 is a major version on our side - is pre-1.0.x react-native something we want to support in our 1.x version?  If we later stop supporting it, we'd have to bump our major version.  (I know nothing of how much 0.57+ is used in react-native, so the answer may just be "yes")

Reply:
The latest react-native version is 0.60c (which is still in beta). I guess for now we would want to support versions from 0.57.0 - 0.59.9 (the most stable versions)

Reply:
Good to know.  Yes, we don't want to advertise compatibility with anything that may break the API (which sub 1.0 versions can do anytime).  Normally this means we stay in the same major version, but if react-native isn't 1.0 yet, we should limit to the most recent version we've tested against, which can be the beta or the stable, whichever you feel more confident about.